src/main.rs

```diff
@@ -172,6 +172,8 @@ fn run(mut config: Config, options: cli::Options) -> Result<(), Box<Error>> {
 
         // Maybe draw the terminal
         if terminal.needs_draw() {
+            // Try to update the position of the input method editor
+            display.update_ime_position(&terminal);
```

Comment:
The IME position is not updated after the cursor position changed on first load. This will only update the IME position after the terminal is dirty.

Current
```
user@linux ~> |
_
```
Expected
```
user@linux ~> |
              _
```
`|` is the current terminal cursor position and `_` is the IME position.

Reply:
Whenever the terminal draws something, it will be marked as dirty. So the ime position should be updated after the cursor is at the position that is currently displayed.

Reply:
But currently in alacritty 0.2.7, that doesn't seem to be the behavior though.

Reply:
There might be a separate issue, however I don't believe that is it.

Please take a look at https://github.com/jwilm/alacritty/issues/2056 for more information.